tools/functional-tester/etcd-tester/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
 	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	failureWrapper := flag.String("failure-wrapper", "", "specify a path of script for enabling/disabling an external fault injector")
```

Comment:
Current name of the option was suggested by @heyitsanthony https://github.com/coreos/etcd/pull/6519#discussion_r80982679 but I think both of it and external-failures are ok. Maybe external-failure would be easy to understand a little bit?